models/mednist_reg/docs/README.md

```diff
@@ -0,0 +1,102 @@
+# MedNIST Hand Image Registration
+
+Based on [the tutorial of 2D registration](https://github.com/Project-MONAI/tutorials/tree/main/2d_registration)
+
+## Downloading the Dataset
+Download the dataset [from here](https://github.com/Project-MONAI/MONAI-extra-test-data/releases/download/0.8.1/MedNIST.tar.gz) and extract the contents to a convenient location.
+
+The MedNIST dataset was gathered from several sets from [TCIA](https://wiki.cancerimagingarchive.net/display/Public/Data+Usage+Policies+and+Restrictions),
+[the RSNA Bone Age Challenge](http://rsnachallenges.cloudapp.net/competitions/4),
+and [the NIH Chest X-ray dataset](https://cloud.google.com/healthcare/docs/resources/public-datasets/nih-chest).
+
+The dataset is kindly made available by [Dr. Bradley J. Erickson M.D., Ph.D.](https://www.mayo.edu/research/labs/radiology-informatics/overview) (Department of Radiology, Mayo Clinic)
+under the Creative Commons [CC BY-SA 4.0 license](https://creativecommons.org/licenses/by-sa/4.0/).
+
+If you use the MedNIST dataset, please acknowledge the source.
+
+
+## Training
+
+Training with same-subject image inputs
+```bash
+python -m monai.bundle run training --config_file configs/train.yaml --dataset_dir "/workspace/data/MedNIST/Hand"
+```
+
+Training with cross-subject image inputs
+```bash
+python -m monai.bundle run training \
+  --config_file configs/train.yaml \
+  --dataset_dir "/workspace/data/MedNIST/Hand" \
+  --cross_subjects true
+```
+
+Training from an existing checkpoint file, for example, `models/model_key_metric=-0.0734.pt`:
+```bash
+python -m monai.bundle run training --config_file configs/train.yaml [...omitting other args] --ckpt "models/model_key_metric=-0.0734.pt"
+```
+
+
+## Inference
+
+The following figure shows an intra-subject (`--cross_subjects false`) model inference results (Fixed, moving and predicted images from left to right)
+
```

Comment:
Here could "cuda:1" be emphasized like other examples below and above (for `5` epochs, for example, `models/model_key_metric=-0.0734.pt`) ?